R/utilities.R

```diff
@@ -2741,45 +2849,52 @@ collate_lymphgen = function(sample_table,
 #' @import tidyverse
 #'
 #' @examples
-#' sample_table <- data.frame (sample_id  = c("09-33003T", "05-18426T", "BLGSP-71-06-00174-01A-01D", "BLGSP-71-22-00347-01A-12E", "SP59282"))
-#' qc_metrics = collate_qc_results(sample_table = sample_table, seq_type_filter = "genome")
+#' qc_metrics = collate_qc_results(sample_table = sample_table)
 #'
 collate_qc_results = function(sample_table,
-                              seq_type_filter = "genome"){
-
+                              seq_type_filter){
+  
   if(! seq_type_filter %in% c("genome", "capture")){
     stop("Please provide a valid seq_type (\"genome\" or \"capture\").")
   }
-
+  
   #get paths
   base = config::get("project_base") #base
   qc_template = config::get("qc_met") #qc metric template
   qc_path = glue::glue(qc_template) #glue wildcards
   icgc_qc_path_full = paste0(base, qc_path) #combine for icgc_dart qc metric path
   gambl_qc = gsub("icgc_dart", "gambl", qc_path) #use gsub to substitute icgc_dart in qc path with gambl
```

Comment:
This is not the standard way we substitute in wildcards. You should look at the other examples where we use glue.

Reply:
Will do!